src/base-container/tests/BaseContainer.test.jsx

```diff
@@ -38,6 +39,20 @@ describe('Base component tests', () => {
       LargeScreen,
     );
 
+    expect(container.querySelector('.banner__image')).toBeDefined();
+  });
```

Comment:
[NIT] please add an empty line after this.

Reply:
We can't add empty lines it shows error  `Block must not be padded by blank lines `

Reply:
I was asking to add empty line after the block. Line after line#43 and before start of next test.